src/App.css

```diff
@@ -0,0 +1 @@
+/*# sourceMappingURL=App.css.map */
```

Comment:
Why do you use `css`. We use `scss`
why is that?

Reply:
third-party extension added, removed

Reply:
and moved the footer to the folder components